src/bindings.rs

```diff
@@ -29,21 +30,15 @@ const TAG_UNDEFINED: i64 = 3;
 const TAG_EXCEPTION: i64 = 6;
 const TAG_FLOAT64: i64 = 7;
 
-/// Free a JSValue.
-/// This function is the equivalent of JS_FreeValue from quickjs, which can not
-/// be used due to being `static inline`.
+
+/// Free a JSValue. (decrease refcount and garbage collect if 0).
 unsafe fn free_value(context: *mut q::JSContext, value: q::JSValue) {
```

Comment:
We can just remove this helper and use the newly exposed `q::FreeValue` instead.